services/uncoolifyService.spec.js

```diff
@@ -75,4 +75,8 @@ describe('uncoolifyService.alphaNumericeName', () => {
   it('should uncoolify doublestruck letters', () => {
     expect(alphaNumericeName('𝕙𝕖𝕝𝕝𝕠')[0]).to.equal('hello');
   });
+
+  it('should uncoolify unicodemix letters', () => {
+    expect(alphaNumericeName('ℌΣ↳↳Φ')[0]).to.equal('hellos');
```

Comment:
hellos?

Reply:
Solved the issue in new commit 
https://github.com/NJACKWinterOfCode/cool-name-api/commit/408b5cb26059e14066492ec89f888a441ab6629b